app/seasonal-planting-guide/page.tsx

```diff
@@ -59,67 +64,70 @@ export default function SeasonalPlantingGuide() {
 
   return (
     <PageContainer>
-      {!selectedUsState ? (
-        <>
-          <p>Please select a US state to view planting information.</p>
-          <StateOptionsContainer>
-            <FilterDropdownSingle
-              name="usState"
-              id="usState"
-              value={selectedUsState}
-              setStateAction={setSelectedUsState}
-              placeholder="US State"
-              options={usStateOptions}
-            />
-          </StateOptionsContainer>
-        </>
-      ) : (
-        <>
-          <HeaderContainer>
-            <SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
-            <FilterContainer>
-              <FilterDropdownSingle
-                name="usState"
-                id="usState"
-                value={selectedUsState}
-                setStateAction={setSelectedUsState}
-                placeholder="US State"
-                options={usStateOptions}
-              />
-
-              <FilterDropdownMultiple
-                value={selectedGrowingSeason}
-                setStateAction={setSelectedGrowingSeason}
-                options={growingSeasonOptions}
-                placeholder="Growing Season"
-              />
+      <HeaderContainer>
```

Comment:
note: in the future, we can consider refactoring by making the header (including the search bar and filters) into a separate component that takes in a list of filters as props??
it's great as it is now, but this refactor might make it slightly more reusable in /view-plants??? just an idea